fraud/bad_encoding.go

```diff
@@ -45,17 +45,17 @@ func CreateBadEncodingProof(
 	}
 }
 
-// Type returns type of fraud proof
+// Type returns type of fraud proof.
```

Comment:
Do these period additions in a separate PR so as not to pollute this one with non-functional changes.

Reply:
These changes were requested in #674, so I decided to add it here.

Reply:
all non-functional changes will be introduced in a separate PR

Reply:
https://github.com/celestiaorg/celestia-node/pull/826